<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QDialog" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Administrator</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sterge Melodie :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Restrictioneaza votarea:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="adm_cmb_sterge">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="adm_cmb_res">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="adm_out">
   <property name="geometry">
    <rect>
     <x>182</x>
     <y>364</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign Out</string>
   </property>
  </widget>
  <widget class="QPushButton" name="adm_sterge">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Sterge</string>
   </property>
  </widget>
  <widget class="QPushButton" name="adm_res">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Restricitoneaza</string>
   </property>
  </widget>
  <widget class="QLabel" name="adm_status">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>364</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : black; 
color : white; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="adm_res_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Scoate Restrictie</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>adm_cmb_sterge</zorder>
  <zorder>adm_cmb_res</zorder>
  <zorder>adm_sterge</zorder>
  <zorder>adm_res</zorder>
  <zorder>adm_status</zorder>
  <zorder>adm_out</zorder>
  <zorder>adm_res_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
